README.md

```diff
@@ -406,6 +406,22 @@ jobs:
       - uses: psf/black@stable
 ```
 
+## Spotless
```

Comment:
A separate section for a single tool that's fairly obscure to the Python community seems like overkill. Maybe we could mention this in the "editor integrations" section?

Reply:
Sure, I'll move it there :). I was looking for an "integrations" or "build tools" section.  We don't quite fit "editor integrations", but we're pretty close.